installer/windows/build_installer.py

```diff
@@ -303,122 +303,153 @@ def create_wxs_file():
             Return='ignore'
         />
 
-        <CustomAction Id="ExtractMartools"
-            Directory="INSTALLDIR"
```

Comment:
we don't need directory property anymore?

Reply:
The "Directory" property isn't compatible with "BinaryKey" and "DllEntry" 
So we needed a workaround, the solution is simple, for "ExtractMartools", we don't need to be inside of the specific dir because the "Expand-Archive" command uses the Zip file we tell him to use, and unzip it where we tell it to unzip it

In case of things that need to be in a specific directory to work, for example Acrylic installation, we use a "Set-Location" (equivalent to `cd` in powershell) to move to the needed directory: https://github.com/massalabs/station/pull/1231/files/f39b276be6daa38f8f498c2450972819061ad758#diff-40085a0c3f8a1d74e1ec5324db618bbced740670ce3324856c05f60f09b4245bR346